site/src/api/api.ts

```diff
@@ -152,8 +151,7 @@ export const getTokens = async (): Promise<TypesGen.APIKey[]> => {
 }
 
 export const deleteAPIKey = async (keyId: string): Promise<void> => {
-  const response = await axios.delete("/api/v2/users/me/keys/" + keyId)
-  return response.data
+  await axios.delete("/api/v2/users/me/keys/" + keyId)
```

Comment:
This function indicates it returns `Promise<void>`, so any returned data is ignored. It's violating [`@typescript-eslint/no-unsafe-return`](https://typescript-eslint.io/rules/no-unsafe-return) anyway so I figured I'd clean it up. 🙂